rpm/inicpp.spec

```diff
@@ -0,0 +1,48 @@
+Name: inicpp
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: C++ parser of INI files.
+License: MIT
+BuildRequires: cmake, make, (gcc-c++ or clang)
+URL: https://github.com/SemaiCZE/%{name}
+Patch0: %{name}-cmake.patch
+Source0: https://github.com/SemaiCZE/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+%description
+Modern, C++14 library for parsing INI files with schema validation.
+
+%prep
+%autosetup
+
+%build
+mkdir build
+cd build
+%cmake ..
+%make_build
+cd ..
+
+%install
```

Comment:
I'd prefer to use `make install DESTDIR=%{buildroot}` here. If there is any problem with installation it should be fixed in `CMakeLists.txt` file.

Reply:
Done, edited CMakeLists.txt to respect INCLUDE_INSTALL_DIR and LIB_INSTALL_DIR defined by fedora's %cmake macro, which point to /usr/include and /usr/lib (or /usr/lib64) respectively.  Also changed the defaults for these to point to /usr/local/include and /usr/local/lib since these are the conventional defaults for user-built applications.